Source/Private/Invoke-DockerCommand.ps1

```diff
@@ -0,0 +1,15 @@
+function Invoke-DockerCommand {
```

Comment:
Not sure this new function is necessary? Unless seems like `Invoke-Command docker "your command"` seems about the same length 😆

Reply:
The length of the command is the same. But one solution (the one I have done) means having this definition in once place. The other one sprinkles hard-coded string values all over the solution.